src/main/java/com/aws/greengrass/logmanager/LogManagerService.java

```diff
@@ -675,7 +663,14 @@ private void processLogsAndUpload() throws InterruptedException {
         }
     }
 
+    public Runnable registerEventStatusListener(Consumer<EventType> callback) {
```

Comment:
Where is this function used?

Reply:
In the Integ test. This is just for allowing test can track the status of service.